commands_config/config_command.go

```diff
@@ -37,6 +37,7 @@ func ConfigAction() func(c *cli.Context) error {
 		setupValue("model", msgForSetupModels)
 		setupValue("openai_api_key_path", "")
 		setupValue("log_level", "")
+		setupValue("gcp.service_account_key_path", "")
```

Comment:
for the future would be nice to separate code change and doc change to the different PRs